cattle/plugins/volmgr/volmgr.py

```diff
@@ -203,6 +203,20 @@ def update_managed_volume(instance, config, start_config):
     start_config['binds'] = new_binds_map
 
 
+def snapshotExists(snapshot_uuid, volume_uuid):
+    volumes = v.list_volumes(volume_uuid)
```

Comment:
Is this always a fast operation?  will this slow down if i have tons of volumes?